Iterable/iterable-web-sdk
# Embedded received event

## JIRA Ticket

* [MOB-7508](https://iterable.atlassian.net/browse/MOB-7508)

## Description

> We have updated the request params accordingly.

## Test Steps

> Begin by launching the sample app, proceed to select the 'embedded' option. Enter your userId/email in the designated field. Subsequently, choose the 1st option for testing and verify the params in request.

[MOB-7508]: https://iterable.atlassian.net/browse/MOB-7508?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
